app/views/merchants/edit.html.erb

```diff
@@ -0,0 +1,8 @@
+<%= form_tag("/merchants/#{@merchant.id}", method: :patch) do %>
+  <p><label> Name: </label><%= text_field_tag :name %></p>
```

Comment:
Rather than the html <label> tag, you can use an ActionViewHelper: `<%= label_tag "Name:" %>` for example